test/integration/maintaining_repo_subscriptions_test.rb

```diff
@@ -34,6 +34,17 @@ def triage_the_sandbox
     assert page.has_content?("mockstar")
   end
 
+  test "clicking an issue leads to a github page" do
+    triage_the_sandbox
+    assert page.has_link?("first test issue", :href => /github.com/)
```

Comment:
What do you think about using the Ruby 1.9 hash syntax here?
```ruby
href: /github.com/
```

Reply:
good idea, should've done that in the first place :smile: